client/src/screens/Client/Manage/AccountingSearch/AccountingSearch.js

```diff
@@ -203,23 +202,35 @@ export default function AccountingSearch() {
                 />
               </Grid>
               <Grid item xs={12} sm={6}>
-                <TextField
+                <KeyboardDatePicker
                   variant="outlined"
                   id="date"
                   label="Date From"
                   value={dateFrom}
                   className={classes.textField}
                   onChange={handleDateChangeFrom}
```

Comment:
You don't need separate function. You can do in the following way: `onChange={(date)=> setDateFrom(date)}`